taxi/urls.py

```diff
@@ -0,0 +1,12 @@
+from django.urls import path
+
+from taxi.views import index
+
+urlpatterns = [
+    path("", index, name="index"),
+    path("manufacturers/", index, name="manufacturers"),
```

Comment:
actually it's not a good idea when you home page opens by the `manufacturers/`, `cars/` or `drivers/` url